### PR TITLE
Add API endpoints to the front-end

### PR DESCRIPTION
**Pull request by Debas-31**
- [x] Added API endpoints to the following feature: Solved https://github.com/sentayhu19/Book-an-appointment-back-end/issues/17
  - User 
     - login
     - registration
  - Motorcycle
     - Add, show, update, and delete
  - Reservation
    - Add, show, update, and delete

### CODE REVIEW

File: src/redux/api.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  },
+  fetchMotors: (success) => {
+    axios.get(`${baseURL}/motocycles`)
+      .then((response) => {
+        success(response);
+      })
+      .catch((error) => {
+        success(error.message);
+      });
+  },
```
Comment (Achilles-Dev) [inline src/redux/api.js]: Please the spelling of `motorcycles` is incorrectly spelled as `motocycles`

File: src/redux/api.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  addMotor: (motor, success) => {
+    axios.post(`${baseURL}/motocycles`, {
+      motors: motor,
+    })
```
Comment (Achilles-Dev) [inline src/redux/api.js]: Again the spelling of `motorcycles` is incorrectly spelled as `motocycles`

**Review by Achilles-Dev — CHANGES_REQUESTED** [review]
Hi @Debas-31 , :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

File: src/redux/api.js (hunk 2 above)
Comment (Debas-31) [reply src/redux/api.js]: Thank you for the feedback; I will implement it :+1:

**Review by Achilles-Dev — APPROVED** [review]
## Status Approved  :green_circle: 
Hello @Debas-31  
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:

**Comment by AyoubM490** [comment]
Good job implementing the changes specified by @Achilles-Dev 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! tada

**Pull request merged**